can/interfaces/pcan/pcan.py

```diff
@@ -268,6 +268,12 @@ def __init__(
 
         self.check_api_version()
 
+        if not isinstance(state, BusState):
```

Comment:
I think this is the wrong place for this. The config value should be converted in `can.util._create_bus_config()`.